
#include "centre_player_scene_replied_handler.h"

///<<< BEGIN WRITING YOUR CODE
///<<< END WRITING YOUR CODE



void CentrePlayerSceneRepliedHandler::EnterScene(entt::entity player,const ::CentreEnterSceneRequest* request,
	::google::protobuf::Empty* response)
{
///<<< BEGIN WRITING YOUR CODE
///<<< END WRITING YOUR CODE

}


void CentrePlayerSceneRepliedHandler::LeaveScene(entt::entity player,const ::CentreLeaveSceneRequest* request,
	::google::protobuf::Empty* response)
{
///<<< BEGIN WRITING YOUR CODE
///<<< END WRITING YOUR CODE

}


void CentrePlayerSceneRepliedHandler::LeaveSceneAsyncSavePlayerComplete(entt::entity player,const ::CentreLeaveSceneAsyncSavePlayerCompleteRequest* request,
	::google::protobuf::Empty* response)
{
///<<< BEGIN WRITING YOUR CODE
///<<< END WRITING YOUR CODE

}


void CentrePlayerSceneRepliedHandler::SceneInfoC2S(entt::entity player,const ::CentreSceneInfoRequest* request,
	::google::protobuf::Empty* response)
{
///<<< BEGIN WRITING YOUR CODE
///<<< END WRITING YOUR CODE

}

